napari/_qt/dialogs/qt_notification.py

```diff
@@ -344,7 +348,13 @@ def show_tb(parent):
                 tbdialog.setLayout(QVBoxLayout())
 
                 text = QTextEdit()
-                text.setHtml(notification.as_html())
+                with warnings.catch_warnings():
```

Comment:
Could you please add a comment about which warning this is silencing? This will help future maintenance in updating API calls etc.

Reply:
Stupid copy and paste from another code part. After the check fixed to not use the warnings filter.